website_demo/README.md

```diff
@@ -0,0 +1,39 @@
+# Wedding Website
```

Comment:
Please remove this file and images that aren't relevant to our website; there's A LOT of them